refresh/templates/common/Application.swift

```diff
@@ -26,18 +26,19 @@ public var basePath = "<%- basepath %>"
 public let health = Health()
 <% } -%>
 <% if (appInitCode.services.length > 0) { -%>
-
```

Comment:
Removing the blank line here will mean the generated app won't have a blank line between the definition of `health` and the start of the `ApplicationServices` class any more; was that the intention?

Reply:
no, it was a mistake. Now fixed